README.md

```diff
@@ -32,47 +34,48 @@ KubePlus provides controls to set per-namespace resource quotas. It also monitor
 ### Upgrades
 
 A running application instance can be updated by making changes to the spec properties of the CRD instance and applying it.
-KubePlus will update that application instance (i.e. helm upgrade of the corresponding helm release). 
+KubePlus will update that application instance (i.e. helm upgrade of the corresponding helm release).
 A new version of an application can be deployed by updating the application Helm chart under the existing Kubernetes CRD or registering the new chart under a new Kubernetes CRD. If the existing Kubernetes CRD object is updated, KubePlus will update all the running application instances (helm releases) to the new version of the application Helm chart.
 
 ### Customization
 
-The spec properties of the Kubernetes CRD wrapping the application Helm chart are the fields defined in the chart’s values.yaml file. Application deployments can be customized by specifying different values for these spec properties. 
-
-
-## Demo
-
-https://github.com/cloud-ark/kubeplus/assets/732525/efb255ff-fc73-446b-a583-4b89dbf61638
+The spec properties of the Kubernetes CRD wrapping the application Helm chart are the fields defined in the chart’s values.yaml file. Application deployments can be customized by specifying different values for these spec properties.
 
-## Getting Started with an example
+## Installation
 
-Let’s look at an example of creating a multi-instance WordPress Service using KubePlus. The WordPress service provider goes through the following steps towards this on their cluster:
+### Manual steps
 
-1. Create cluster or use an existing cluster. For testing purposes you can create a [minikube](https://minikube.sigs.k8s.io/docs/) or [kind](https://kind.sigs.k8s.io/) cluster:
+* Create cluster or use an existing cluster. For testing purposes you can create a [minikube](https://minikube.sigs.k8s.io/docs/) or [kind](https://kind.sigs.k8s.io/) cluster:
 
    `minikube start`
 
    or
 
    `kind create cluster`
 
-2. Unzip KubePlus plugins and set up the PATH
+* Set the Namespace in which to deploy KubePlus
 
-   ```
+   `export KUBEPLUS_NS=default`
+
+* Unzip KubePlus plugins and set up the PATH
+
+   ```sh
    wget https://github.com/cloud-ark/kubeplus/raw/master/kubeplus-kubectl-plugins.tar.gz
    tar -zxvf kubeplus-kubectl-plugins.tar.gz
    export KUBEPLUS_HOME=`pwd`
    export PATH=$KUBEPLUS_HOME/plugins:$PATH
    kubectl kubeplus commands
    ```
 
-3. Set the Namespace in which to deploy KubePlus
+* Set the Namespace in which to deploy KubePlus
```

Comment:
Repeated (lines 56-58 already set KUBEPLUS_NS). Delete lines 56-58.

Reply:
Updated the README to inculcate the changes.